packages/server/src/centrifuge-client/centrifuge.module.ts

```diff
@@ -8,10 +8,6 @@ function checkNodeEnvironment() {
     case 'test': {
       return new MockCentrifugeService();
     }
-    case 'functional': {
```

Comment:
This was also generating typescript error. I assumed we are not using this so I removed it. If we want to add e2e test we can figure it out then